pkg/planner/core/operator/logicalop/logical_cte.go

```diff
@@ -175,7 +175,7 @@ func (p *LogicalCTE) PushDownTopN(topNLogicalPlan base.LogicalPlan, opt *optimiz
 // RecursiveDeriveStats inherits BaseLogicalPlan.LogicalPlan.<10th> implementation.
 
 // DeriveStats implements the base.LogicalPlan.<11th> interface.
-func (p *LogicalCTE) DeriveStats(_ []*property.StatsInfo, selfSchema *expression.Schema, _ []*expression.Schema, _ [][]*expression.Column) (*property.StatsInfo, error) {
+func (p *LogicalCTE) DeriveStats(_ []*property.StatsInfo, selfSchema *expression.Schema, _ []*expression.Schema, _ [][]*expression.Column, _ bool) (*property.StatsInfo, error) {
```

Comment:
```suggestion
func (p *LogicalCTE) DeriveStats([]*property.StatsInfo, selfSchema *expression.Schema, []*expression.Schema, [][]*expression.Column, bool) (*property.StatsInfo, error) {
```

Reply:
the func sig seems require this underscore